src/content_handlers/apt_handler/tests/sample_apt_manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "du-agent-update",
+    "version": "0.7.0",
+    "packages": [
+        {
+            "name": "deviceupdate-agent",
+            "version": "0.7.0~public~preview"
+        }
+    ],
+    "agentRestartRequired": true
+}
```

Comment:
how will someone publishing an agent update package know to include this?  I'm concerned that the connection is non-obvious, and we'll end up publishing packages without restartrequired.

Reply:
We will update the documents and communicate to customer the best we can. 

---
In reply to: [596184694](https://github.com/Azure/iot-hub-device-update/pull/17#discussion_r596184694) [](ancestors = 596184694)